templates/add.html

```diff
@@ -0,0 +1,14 @@
+{% extends "layout.html" %}
+{% block title %}Add To-Do Item{% endblock %}
+
+{% block content %}
+    <h1>Add Item</h1>
+    <form action="" method="post" novalidate>

```

Comment:
You definitely want to enable client side validation as there will be no hint to the user if something goes wrong